migrate/playbooks/rds-timescale-live-migration.md

```diff
@@ -0,0 +1,410 @@
+---
+title: Migrate from RDS to Timescale using live migration
+excerpt: Migrate from a PostgreSQL database in AWS RDS to Timescale using live migration
+products: [cloud]
+keywords: [data migration, postgresql, RDS]
+tags: [migrate, AWS, RDS, low-downtime, pg_dump, pgcopydb]
+---
+
+import GettingHelp from "versionContent/_partials/_migrate_dual_write_backfill_getting_help.mdx";
+import SourceTargetNote from "versionContent/_partials/_migrate_source_target_note.mdx";
+import StepOne from "versionContent/_partials/_migrate_dual_write_step1.mdx";
+import LiveMigrationRoles from "versionContent/_partials/_migrate_live_migration_rds_roles.mdx";
+import DumpDatabaseRoles from "versionContent/_partials/_migrate_dual_write_dump_database_roles.mdx";
+import DumpSourceSchema from "versionContent/_partials/_migrate_dump_source_schema.mdx";
+
+# Migrate from AWS RDS to Timescale using live migration with low-downtime
+This document provides a step-by-step guide to migrating a database from an AWS
+RDS Postgres instance to Timescale using our [Live migration] strategy to achieve
+low application downtime (on the order of minutes).
+
+Live migration's replication mechanism is fundamentally based on Postgres' logical
+decoding feature. However, for the purpose of this guide, an in-depth understanding
+of logical decoding is not necessary. It is important to note that the "Live migration"
+strategy is significantly more complicated than a migration using [pg_dump/pg_restore].
```

Comment:
I would suggest removing this line.

Why is it considered significantly complicated? Aren't our scripts and Docker image designed to simplify this process? Mentioning upfront that it is significantly complicated may discourage users from proceeding right from the beginning. :)

Reply:
The scripts/docker images do help and ease the migration, but in this case, the user has to manually execute the commands, rather than a script. I will rephrase to avoid any discouragement.